roles/KubevirtNodeLabeller/defaults/main.yml

```diff
@@ -2,11 +2,7 @@
 # defaults file for KubevirtNodeLabeller
 wait: true
 
-kubevirt_node_labeller_image: "kubevirt-node-labeller"
-kubevirt_node_labeller_tag: "v0.0.5"
```

Comment:
I would actually keep tags here as well to make sure it all works even without the _default file. But set them to "latest" so we do not have to maintain the versions here.